plugins/advanced_regex/plugin.spec.yaml

```diff
@@ -4,23 +4,43 @@ products: [insightconnect]
 name: advanced_regex
 title: Advanced Regex
 description: Perform advanced regular expression operations on a string using Python regex
-version: 1.0.3
+version: 1.0.4
+connection_version: 1
 support: community
+supported_versions: ["2024-10-01"]
+fedramp_ready: true
 vendor: rapid7
 status: []
 resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/advanced_regex
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
 tags:
-- data manipulation
-- utility
+  - data manipulation
+  - utility
 hub_tags:
   use_cases: [data_utility]
   keywords: [data manipulation, utility]
   features: []
 enable_cache: true
 language: python
-
+sdk:
+  type: slim
+  version: 6.1.3
+  user: nobody
+key_features:
+  - Data extraction
+  - Search and replace text
+  - Split text
+version_history:
+  - "1.0.4 - Initial updates for fedramp compliance | Updated SDK to the latest version"
+  - "1.0.3 - Update to make replace string non-required"
+  - "1.0.2 - Update to v4 Python plugin runtime | Add example inputs"
+  - "1.0.1 - New spec and help.md format for the Extension Library"
+  - "1.0.0 - Initial plugin"
+links:
+  - "[Python Regex](https://docs.python.org/library/re.html)"
```

Comment:
you can put the repository link like below:

`- "[Advanced Regex](https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/advanced_regex)"`

Reply:
Imo we can skip it for now